src/dune_engine/action_builder.mli

```diff
@@ -54,8 +54,8 @@ with type 'a build := 'a t
     information. *)
 val set_file_system_accessors : file_exists:(Path.t -> bool) -> unit
 
-(** Add a set of targets to a build description, turning a target-less [Build.t]
-    into [Build.With_targets.t]. *)
+(** Add a set of targets to a build description, turning a target-less
```

Comment:
The comments in this module say "build description" when referring to `t` which may now be confusing. Perhaps, we should replace "build description" with "action builder".

Reply:
Agreed, I updated all the comments mentioning "build description"